ownership-chain/pallets/laos-evolution/src/lib.rs

```diff
@@ -283,3 +325,20 @@ where
 
 	Ok(CollectionId::from_be_bytes(id_bytes))
 }
+
+/// Checks if the given account is the owner of the specified collection.
```

Comment:
sorry again, but this can/should be function of the `Pallet`, no reason to make it generic